README.md

```diff
@@ -36,6 +39,14 @@ ws.end(function() {
 })
 ```
 
+## API
+
+See https://github.com/maxogden/abstract-blob-store and in addition:
+
+### store.list(cb)
```

Comment:
You'll need to patch `blob-store-replication-stream` if you want this to work.

I suggest the patch checks for both `list` and `_list` and calls whatever it finds (`list` preferred), for backwards compatibility.